packages/webpack-cli/lib/utils/arg-parser.js

```diff
@@ -63,6 +63,11 @@ function argParser(options, args, argsOnly = false, name = '', helpFunction = un
             if (!option.multiple) {
                 // Prevent default behavior for standalone options
                 parserInstance.option(flagsWithType, option.description, option.defaultValue).action(() => {});
+                if (option.negative) {
+                    // commander requires explicitly adding the negated version of boolean flags
+                    const negatedFlag = `--no-${option.name}`;
+                    parserInstance.option(negatedFlag, `negates ${option.name}`);
+                }
```

Comment:
Add the check here so that `--no-stats` is recognized.
https://github.com/webpack/webpack-cli/blob/7489e639d13e8b89690a50595eb48214e9cdb1d9/packages/webpack-cli/lib/utils/arg-parser.js#L74-L76

Reply:
`--no-stats` is not recognized with the changes made as part of afadcd1

Reply:
fixed